app/src/main/java/org/oppia/app/player/state/CustomCheckbox.kt

```diff
@@ -0,0 +1,89 @@
+package org.oppia.app.player.state
+
+import android.content.Context
+import android.text.Html
+import android.text.Spannable
+import android.text.Spanned
+import android.view.View
+import android.widget.CheckBox
+import android.widget.CompoundButton
+import android.widget.TextView
+import android.widget.Toast
+import androidx.core.content.ContextCompat
+import org.oppia.app.R
+import org.oppia.util.data.UrlImageParser
+import android.content.res.ColorStateList
+import android.os.Build
+import android.view.Gravity
+
+// TODO(#190): Move this to a custom.inputinteractionview.
+/** Custom Checkbox for MultipleSelectionInputInteractionView. */
+class CustomCheckbox : CheckBox {
+  private var mContext: Context
+  private var optionContents: String? = null
+
+  constructor(context: Context, optionContents: String) : super(context) {
+    this.mContext = context
+    this.optionContents = optionContents
+    initViews()
+  }
+
+  //update default attributes of MultipleSelectionInputInteractionView here
+  fun initViews() {
```

Comment:
Rather than calling `initViews()` inside `init` better implement the entire code of `initViews` inside `init` itself and remove the `initViews method`

Reply:
added entire code to init()